File: Forest fire control/Forest fire control.Data/Model/ApplicationModel.cs
using Forest_fire_control.Data.Enums;
using Forest_fire_control.Data.Models;
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Text;

namespace Forest_fire_control.Data.Model
{
    public class ApplicationModel
    {
        public string UserEmail{ get; set; }
        public ObservationSiteModel ObservationSite { get; set; }
        public DateTime Data { get; set; }
        public string Description { get; set; }
        public IncedentStatusEnum Status { get; set; }
    }
}

File: Forest fire control/Forest fire control/ClientApp/src/app/models/application.model.ts
import { IncedentStatusEnum } from "./emums/incident-status.emum";
import { ObservationSite } from "./observation.model";

export interface Application {

    userEmail: string ;
    observationSite: ObservationSite;
    data: Date;
    description: string;
    status: IncedentStatusEnum;
  }